File: testdata/jennies/rawtypes/intersections/JavaRawTypes/intersections/Intersections.java
package intersections;

import externalPkg.AnotherStruct;

public class Intersections extends SomeStruct, AnotherStruct {
    public String fieldString;
    public Integer fieldInteger;
    public Intersections() {}

    public Intersections(String fieldString,Integer fieldInteger) {
        this.fieldString = fieldString;
        this.fieldInteger = fieldInteger;
    }
    
}

File: testdata/jennies/rawtypes/intersections/CSharpRawTypes/src/Grafana/Foundation/Intersections/Intersections.cs
namespace Grafana.Foundation.Intersections;

using Grafana.Foundation.ExternalPkg;

public class Intersections : SomeStruct, AnotherStruct
{
    public string FieldString;
    public int FieldInteger;

    public Intersections()
    {
        this.FieldString = "hello";
        this.FieldInteger = 32;
    }

    public Intersections(string fieldString, int fieldInteger)
    {
        this.FieldString = fieldString;
        this.FieldInteger = fieldInteger;
    }
}
